packages/rsc/src/stream-ui/stream-ui.tsx

```diff
@@ -262,7 +266,7 @@ functionality that can be fully encapsulated in the provider.
     model.doStream({
       ...prepareCallSettings(settings),
       ...prepareToolsAndToolChoice({
-        tools,
```

Comment:
this is necessary until we do the internal bundle merges or either the provider-utils split of tools

Reply:
why? doesn't rsc depend on `ai`?

Reply:
it does at the top